server.js

```diff
@@ -2,14 +2,14 @@
 const morgan = require('morgan');
 const express = require('express');
 const path = require('path');
-const redis = require('redis');
+// const redis = require('redis');
```

Comment:
Why did you get rid of Redis?

Reply:
because i'm caching on my proxy server rather than my service.  it allows to respond with cache sooner with less API calls and avoid layered caching that can lead to data inconsistency.